docker/server/Dockerfile

```diff
@@ -70,7 +70,6 @@ WORKDIR /home/$USERNAME
 RUN wget https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/23.0.0.8/openliberty-javaee8-23.0.0.8.zip && \
         unzip openliberty-javaee8-23.0.0.8.zip
 
-ADD https://api.github.com/repos/CMU-cabot/MapService/git/refs/heads/cabot-hokoukukan_2018 MapService-version.json
```

Comment:
This line should be here to deal with the cacheing issue with the git clone line.